rest-api/src/main/java/gov/cms/qpp/conversion/api/logging/AttachmentHashPartConverter.java

```diff
@@ -30,7 +27,7 @@ public String convert(ILoggingEvent event) {
 		try {
 			hashPart = getHashPart();
 		} catch (IOException | ServletException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)

Reply:
Nope, bad idea.  This is in a logging support class and doing either of this will destroy the output of our logs.  Go home, you're drunk.  😝